pkg/validation/projects.go

```diff
@@ -23,16 +23,21 @@ func ValidateStarterProjects(starterProjects []v1alpha2.StarterProject) error {
 
 		switch len(gitSource.Remotes) {
 		case 0:
-			errString += fmt.Sprintf("\nstarterProject %s should have at least one remote", starterProject.Name)
+			starterProjectErr := fmt.Errorf("\nstarterProject %s should have at least one remote", starterProject.Name)
+			newErr := resolveErrorMessageWithImportAttributes(starterProjectErr, starterProject.Attributes)
+			errString += newErr.Error()
```

Comment:
I think we should accumulate errors in `[]string` and use `strings.Join(errList, "\n")`

Reply:
updated. Also update all error concatenation to use []string & string join